main.go

```diff
@@ -73,15 +73,16 @@ func eventHandler(evt interface{}) {
 		quotedVideo := quotedMsg.GetVideoMessage()
 		quotedText := extended.GetText()
 		messageSender := eventInfo.Info.Sender.User
+		groupMessage := eventInfo.Info.IsGroup
 
 		imageMatch := captionIsCommand(eventInfo.Message.GetImageMessage().GetCaption())
 		videoMatch := captionIsCommand(eventInfo.Message.GetVideoMessage().GetCaption())
 		// check if quoted message with correct caption references media
 		quotedMatch := captionIsCommand(quotedText) &&
 			(quotedImage != nil || quotedVideo != nil)
-
-		if imageMatch || videoMatch || quotedMatch {
-			if quotedMatch {
+		isPrivateMedia := (quotedImage != nil || quotedVideo != nil) && !groupMessage
+		if imageMatch || videoMatch || quotedMatch || isPrivateMedia {
+			if quotedMatch || isPrivateMedia {
```

Comment:
you don't need to add isPrivateMedia on this line... this line is for when someone quotes a previously sent media